R/translate-sql-cut.R

```diff
@@ -0,0 +1,61 @@
+sql_cut <- function(x, breaks, labels = NULL, right = TRUE) {
```

Comment:
Worth implementing `include.lowest`?

Reply:
The case work was a bit annoying but it makes sense after all I guess.